### PR TITLE
CB-13353 : added save-exact as an option

### PR DESCRIPTION
**Pull request by audreyso**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -72,6 +73,11 @@ module.exports = function (target, dest, opts) {
                 fetchArgs.push('--save');
             }
 
+            if (opts.save_exact) {
```
Comment (stevengill) [inline index.js]: So I think this implementation could allow both `--save` and `--save-exact` to be added to `fetchArgs` if `opts.save = true` and `opts.save_exact = true`. 

I think we should only have one or the other. 

```
 if (opts.save_exact) {
    events.emit('verbose', 'saving exact');
    fetchArgs.push('--save-exact');
} else if (opts.save) {
    events.emit('verbose', 'saving');
    fetchArgs.push('--save');
 }
```

**Comment by audreyso** [comment]
@stevengill Thanks for your feedback- made changes and also cleaned up the test file. Let me know if there's anything else!

File: index.js [hunk 2]
```diff
@@ -75,6 +76,14 @@ module.exports = function (target, dest, opts) {
                 fetchArgs.push('--no-save');
             }
 
+            if (opts.save_exact) {
```
Comment (stevengill) [inline index.js]: This if-else (lines 79-85) needs to be combined with the if-else right above it (lines 71-77)

**Comment by stevengill** [comment]
merge it in!

**Pull request merged**